README.md

```diff
@@ -269,3 +269,84 @@ $ mkdocs serve
 ```
 
 then open your browser on: `http://127.0.0.1:8003/`
+
+## Dockerisation
+
+You can build a docker for `mse home` as follow:
+
+```console
+$ docker build -t mse-home . 
+```
+
+Then run it:
+
+```console
+$ # You have to create /mnt/workspace
+$ sudo mkdir /mnt/workspace
+$ # Scaffold a project
+$ docker run -v /mnt/workspace:/mnt/workspace \
+             -v /var/run/docker.sock:/var/run/docker.sock \
+            --network=host \
```

Comment:
is it really needed as everything could happen without any internet connection?

Reply:
Actually we query localhost... so we need to use `--network=host`

Reply:
yes later just when we spawn no?

Reply:
true: not when we scaffold. I tried to write the same args everywhere to avoid the reader missing an arg on a specific command. But I can create lite commands if you prefer when needed

Reply:
yeah, just add the `--network=host` for spawn only